json-to-pot.js

```diff
@@ -94,7 +93,7 @@ const BASE_URL = 'https://github.com/WordPress/openverse-frontend/blob/main'
  */
 const getRefComment = (keyPath) => {
   const keyValue = PARSED_VUE_FILES.find((k) => k.path === keyPath)
-  return keyValue ? `\n#: ${BASE_URL}${keyValue.file}#L${keyValue.line}` : ''
+  return keyValue ? `\n#: ${keyValue.file}#L${keyValue.line}` : ''
```

Comment:
```suggestion
  return keyValue ? `\n#: ${keyValue.file}:${keyValue.line}` : ''
```
Missed the change from GitHub ..#L..  to ...:... for the line number.